tool/github-action/sass-spec-ref.sh

```diff
@@ -7,15 +7,20 @@
 # run. If we're running specs for a pull request which refers to a sass-spec
 # pull request, we'll run against the latter rather than sass-spec master.
 
-GITHUB_REF=${PR_REF:-$CURRENT_REF}
-if [[ "$GITHUB_REF" == refs/heads/feature.* ]]; then
-  default="${GITHUB_REF:11}"
+if [ -z "$PR_BRANCH" ]; then
+  current_branch="${CURRENT_REF:11}"
```

Comment:
Can you add a comment indicating what string exactly the `:11` is slicing off here?

Reply:
done